bittensor/_neuron/text/core_server/__init__.py

```diff
@@ -406,7 +411,7 @@ def run(
                     except Exception as e:
                         logger.error('Failure setting weights on chain with error: {}', e)
 
-    def synapse_check(self, synapse, hotkey):
+    def synapse_check(self, synapse, hotkey, inputs_x=None):
```

Comment:
We should expand the checks on `TEXT_LAST_HIDDEN_STATE` as well; it currently only checks stake, but it requires a significant amount of bandwidth to send the response. (e.g. `float32 * 32 * 256 * 1024` is 32MB to upload)

Reply:
Good call; just added this